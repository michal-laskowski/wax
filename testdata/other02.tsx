export function other02() {
    return (
        <div>
            {[["a", "b"]]}
            <c>d</c>
            {["e", ["f"], [["g"]]]}
        </div>
    );
}