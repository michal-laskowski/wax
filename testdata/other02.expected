<div>
    ab
    <c>d</c>
    efg
</div>